src/components/templates/Playa/Playa.tsx

```diff
@@ -6,15 +6,13 @@ import React, {
   useMemo,
```

Comment:
[FYI] most of the import changes here are just due to it being sorted, main changes was the selectors + related imports